### PR TITLE
Pid D100 request for GLCD Driver

### PR DESCRIPTION
**Pull request by berkuteagle**

### CODE REVIEW

File: 1209/D100/index.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+layout: pid
+title: GLCD Driver
+owner: Dwarfs Inc.
```
Comment (Arachnid) [inline 1209/D100/index.md]: The value in this line needs to match the directory name of your org - eg, 'Dwarfs' in this case.

File: 1209/D100/index.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+license: MIT
+site: http://dwarfs.pw
+source: https://github.com/berkuteagle/glcd2usb
```
Comment (Arachnid) [inline 1209/D100/index.md]: Please add a LICENSE file to your repository.

Also, your code doesn't actually appear to use USB. Can you clarify why you need a PID?

**Comment by berkuteagle** [comment]
Work on the project started recently. In the near future we will add electronic circuits and code USB-side. Retry the request later, when everything is ready.

**Comment by Arachnid** [comment]
Okay. Please open a new PR when you actually need a PID.

**Pull request closed without merging**